Zend/zend_compile.c

```diff
@@ -6306,13 +6282,11 @@ void zend_compile_class_decl(zend_ast *ast, zend_bool toplevel) /* {{{ */
 					return;
 				}
 			}
-		} else {
-			if (EXPECTED(zend_hash_add_ptr(CG(class_table), lcname, ce) != NULL)) {
+		} else if (EXPECTED(zend_hash_add_ptr(CG(class_table), lcname, ce) != NULL)) {
```

Comment:
Here and in a few other places, the code below should be de-indented by one level after this change.